pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -506,9 +506,10 @@ class Build {
             if (cleanWorkspace) {
                 try {
                     if (buildConfig.TARGET_OS == "windows") {
-                        // Softlayer machines struggle to clean themselves
+                        // Windows machines struggle to clean themselves, see:
                         // https://github.com/AdoptOpenJDK/openjdk-build/issues/1855
-                        context.sh(script: "rm -rf C:/workspace/openjdk-build/workspace/build/src/build/*/jdk/gensrc")
+                        // https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1419
+                        context.sh(script: "rm -rf ${WORKSPACE}/workspace/build/src/build/*/jdk/gensrc")
```

Comment:
```suggestion
                        context.sh(script: "rm -rf ${WORKSPACE}/build/src/build/*/jdk/gensrc")
```

Reply:
I had a look at the build scripts for both the `azure` and `softlayer` machines;
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-windows-x64-openj9/670/consoleFull
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-windows-x64-hotspot/660/consoleFull

Specifically this bit: (and the equivalent in the openj9 log)
```
00:32:00  Running on build-azure-win2012r2-x64-1 in J:\jenkins\workspace\build-scripts\jobs\jdk11u\jdk11u-windows-x64-hotspot
[Pipeline] {
[Pipeline] echo
00:32:00  checking windows
[Pipeline] echo
00:32:00  changing J:/jenkins/tmp
[Pipeline] ws
00:32:00  Running in J:/jenkins/tmp
```
Doesn't that path `J:/jenkins/tmp` refer to the `$WORKSPACE` , without the `workspace` directory in it?

Reply:
`$WORKSPACE` always points to `workspace/` I believe

Reply:
Alright, lets test it